spec/factories/profiles.rb

```diff
@@ -8,6 +8,13 @@
 
     trait :active do
       active { true }
+      # `activated_at` will be always defined for active profiles,
+      # but it is not defined here so tests will not rely on it
+      # instead of the `active` boolean above as they should
```

Comment:
Recommend adding `activated_at` and `verified_at` fields to accurately reflect what's required in the code.